core/src/main/java/com/graphhopper/storage/GraphHopperStorage.java

```diff
@@ -60,6 +60,10 @@ public GraphHopperStorage(List<CHProfile> chProfiles, Directory dir, EncodingMan
     }
 
     public GraphHopperStorage(List<CHProfile> chProfiles, Directory dir, EncodingManager encodingManager, boolean withElevation, boolean withTurnCosts) {
+        this(chProfiles, dir, encodingManager, withElevation, withTurnCosts, -1);
+    }
+
+    public GraphHopperStorage(List<CHProfile> chProfiles, Directory dir, EncodingManager encodingManager, boolean withElevation, boolean withTurnCosts, int segmentSize) {
```

Comment:
I think there should be a builder for this graph to get rid of these constructor variations. We have `GraphBuilder`, but honestly I almost never use it. I think it currently does 'too much', like its doing more than just being a builder, which is maybe good for a top-level user, but for internal code, tests etc. I almost always prefer using the constructor directly. WDYT shall we create another (pure) builder, or adjust the old `GraphBuilder`, or neither ?

Reply:
Update: I found a compromise for `GraphBuilder`, its now a 'dumb' builder but it has some convenience setters like `.setMmap()`. The only thing I do not like now is that `build()` does not call `create()`. I think this should be the 'default'. I would just allow passing a negative value to `setBytes()` in which case the graph will not be created (just like `build()` does at the moment.